src/frontend/Sider/Behandling/Inngangsvilkår/Aktivitet/BrukCell.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { CheckmarkIcon } from '@navikt/aksel-icons';
+import { Button, Tag } from '@navikt/ds-react';
+
+import { useInngangsvilkår } from '../../../../context/InngangsvilkårContext';
+import { Registeraktivitet } from '../../../../typer/registeraktivitet';
+import { Aktivitet } from '../typer/aktivitet';
+
+function erAktivitetLagtTil(aktiviteter: Aktivitet[], aktivitetFraArena: Registeraktivitet) {
+    return aktiviteter.some((aktivitet) => aktivitetFraArena.id === (aktivitet.kildeId as string));
+}
+
+export function BrukCell({
```

Comment:
Prøver å komme på et bedre navn her. Cell er ikke så veldig forklarende. Tenkte først `BrukKnapp` men er jo ikke bare en knapp så vet ikke om det er bra nok

Reply:
Vi har forresten tidligere sagt at vi bruker const over function, og så har det vært noe diskusjoner på om det burde bli noe annet, men ikke landet på noe. Syns fortsatt dette burde landes før vi evt tar i bruk function mer enn det alt er gjort

Reply:
Vi kan skrive om til `const` ☺️

Reply:
Vi startet også med `BrukKnapp` men mente det var feil. Vi gikk over til `BrukCell` siden det er en celle i en tabell, men ikke sikkert det er bedre. Hva med `BrukAktivitetFraArena`?

Reply:
Hva med å kalle `BrukCell`-komponenten det den faktisk er: `DynamiskKnappForAktiviteterFraArena` ?
Den er dynamisk fordi den endrer utseende: Den blir om til checkmark etter aktiviteten er lagt til, og vil bli om til en Knapp igjen dersom man fjerner aktiviteten etterpå...

Reply:
Bra forslag @evilppo. Jeg er glad i beskrivende navn, men føler kanskje det blir litt langt. Vil ha litt mer "snappy" kode 😉 

@sarahjelle argumenterte med at vi kan kalle den `BrukAktivitetKnapp`. Selv om det strengt tatt ikke er en knapp etter man har brukt aktiviteten, kan man se på det som en deaktivering eller disabeling av knappen. Jeg kjøper den argumentasjon,  så tenker vi går for `BrukAktivitetKnapp`.

Reply:
`BrukAktivitetKnapp` gir også mening, ja: Når man "bruker" noe så indikerer det også en form for endring